bertytech/content/faq/10-Will the app hide ip addresses and metadata?/index.md

```diff
@@ -4,6 +4,10 @@ id: 10
 private: true
 ---
 
-It will mainly depend on the context (relay or not, Bluetooth or not, Luck, 3G, NAT, WiFi, TEC). In the current configuration, it's not hidden by tor for example, but by "noisy" things (P2P). This already limits the ip address leak, and especially the certainty that this ip address is this exact person or device.
-We then plan to use something like i2p/tor or some other onion network technology to make it possible to hide everything, including IP addresses. 
-For metadata: we're doing our best efforts to work on that right now, but even here, it will get better over time.
+Today, IP addresses are hidden by a set of "noise" thanks to P2P. Moreover, it will depend on the context (relay or not, Bluetooth or not, luck, 3G, NAT, WiFi, TEC). This already limits IP address leakage, and especially the certainty that this IP address is indeed that of this person or device.
+
+In the future, we then plan to use something like i2p/tor or another onion network technology to allow hiding everything, including IP addresses. 
+
+In addition, metadata collection is greatly minimized. Instead of a directory linking public keys to personal data such as telephone numbers, the [Berty Protocol](https://berty.tech/docs/protocol/) uses a combination of TOTP and a public key to generate rendezvous point addresses and register its users on IPFS, who can later be contacted by peers wishing to communicate.
```

Comment:
It's a lot better :)

> Instead of a directory linking public keys to personal data such as telephone numbers, the [Berty Protocol](https://berty.tech/docs/protocol/) uses a combination of TOTP ...

it's very ambiguous, the TOTP is clearly something we made against leaking metadata but it's not something related to a phone number but something to make volatile addresses

phone (not telephone) numbers are simply replaced by the public keys

and yes TOTP is used to generate rendezvous address that will change over time

the phrase "register on IPFS" is not really something that exists, we don't write anything on IPFS when someone creates an account, especially we don't push an account somewhere, the account is only on your phone, what we do is to become available on the libp2p DHT if someone knock-knock on our TOTP-based rendezvous-point

I suggest you to continue on hackmd to avoid opening a new PR each time we see a detail